bridge_adaptivity/module/templates/module/widgets/select_policy.html

```diff
@@ -0,0 +1,15 @@
+{% load bootstrap3 %}
+<div style="width:100%;">
+  <select style="width:50%; display:inline-block; " class="policy_select form-control" name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %}>
+  {% for group_name, group_choices, group_index in widget.optgroups %}{% if group_name %}
```

Comment:
Please make this code be more readable by splitting into separate logical lines.